test/tests/mesh_modifiers/sidesets_between_subdomains/between_vector_blocks.i

```diff
@@ -0,0 +1,27 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  nz = 0
+  zmin = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[MeshModifiers]
+  [./subdomain_id]
+    type = AssignElementSubdomainID
+    subdomain_ids = '0 0
+                     1 2'
+  [../]
+  [./extrude]
```

Comment:
The word "blocks" is going away in the future in MOOSE (see https://github.com/idaholab/moose/issues/6968 ).

Try to prefer using the word `subdomain` when possible.